lib/services/role.go

```diff
@@ -354,6 +354,18 @@ func ApplyTraits(r types.Role, traits map[string][]string) types.Role {
 			r.SetDatabaseLabels(condition, applyLabelsTraits(inLabels, traits))
 		}
 
+		options := r.GetOptions()
+		for i, ext := range options.CertExtensions {
+			vals, err := ApplyValueTraits(ext.Value, traits)
+			if err != nil && !trace.IsNotFound(err) {
+				log.Debugf("Error applying trait to cert_extensions.value: %v", err)
```

Comment:
Shouldn't this be logged as warning at least? Debug message starting with "Error" is a bit confusing in my opinion.